app/assets/stylesheets/notes.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the notes controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
We should add some SCSS styles here (try to use some SCSS feature), to test out whether these get compiled into a final CSS file in production. Feel free to make the design terrible (pink, green, whatever), so that it HAS to be changed.

Reply:
I added some styling with basic scss elements. Is that enough or did you have some more complicated features in mind that should be used?